src/main/java/dk/kb/present/webservice/ContextListener.java

```diff
@@ -67,6 +68,8 @@ public void contextInitialized(ServletContextEvent sce) {
             String configFile = (String) ctx.lookup("java:/comp/env/application-config");
             //TODO this should not refer to something in template. Should we perhaps use reflection here?
             ServiceConfig.initialize(configFile);
+            // Early initialization of HoldbackDatePicker
+            HoldbackDatePicker.init();
```

Comment:
A few lines down. end bracked } for method not formatted correct. Move to next line. I know it is not part of this review...